docs/reference/data-streams/lifecycle/tutorial-manage-new-data-stream.asciidoc

```diff
@@ -0,0 +1,148 @@
+[role="xpack"]
+[[tutorial-manage-new-data-stream]]
+=== Tutorial: Create a data stream with a lifecycle
+
+preview::[]
+
+To create a data stream with a built-in lifecycle, follow these steps:
+
+. <<create-index-template-with-lifecycle>>
+. <<create-data-stream-with-lifecycle>>
+. <<retrieve-lifecycle-information>>
+
+[discrete]
+[[create-index-template-with-lifecycle]]
+==== Create an index template
+
+A data stream requires a matching <<index-templates,index template>>. The same way you use index
+templates for mappings and index settings you can use them to configure the data stream lifecycle, the respective
+template is called `lifecycle`. You can define an index template that sets a lifecycle as follows:
+
+* Include the `data_stream` object to enable data streams.
+
+* Define the lifecycle in the template section or include a composable template that defines the lifecycle.
+
+* Use a priority higher than `200` to avoid collisions with built-in templates.
+See <<avoid-index-pattern-collisions>>.
+
+You can use the <<indices-put-template,create index template API>>.
+
+[source,console]
+--------------------------------------------------
+PUT _index_template/my-index-template
+{
+  "index_patterns": ["my-data-stream*"],
+  "data_stream": { },
+  "priority": 500,
+  "template": {
+    "lifecycle": {
+      "data_retention": "7d"
+    }
+  },
+  "_meta": {
+    "description": "Template for with data stream lifecycle"
```

Comment:
"Template for with" - is there a word missing or is "for" extra?

Reply:
"for" is extra.